### DISCUSSION TITLE
Accès "impossible" au portail web OMR

### DISCUSSION THREAD

**Discussion by Altarus1** [Ysurac/openmptcprouter#2067] [Q&A]
Bonjour,

OMR est installé sur RPi4, est configuré et fonctionne correctement depuis plusieurs jours maintenant. j'ai voulu me rendre sur OMR pour vérifier la bande passante utilisée et quelques log. J'arrive bien sur la page de login d'OMR, mais impossible de passer l'authentification avec le compte utilisateur "root" (il n'y a pas d'autres comptes créés). Le mot de passe est le bon, je peux accéder en SSH à la RPi et vérifier les fichiers de config/log en mode console.
J'ai probablement fait un setup dans OMR qu'il ne fallait pas (je pense bien avoir coché une option pour refuser l'accès root, je sais pas si c'est celle là, j'avais cru comprendre que ce serait pour l'accès SSH, et non pas le portail OMR).
Comment modifier le paramètre "bloquer accès root" à partir de la console ?
Merci d'avance

**Comment by Ysurac** [comment]
Peut être dans /etc/config/dropbear, donc via uci : uci show dropbear et pour modifier uci set....

**Reply by Altarus1** [reply]
Merci Yannick pour la réponse, mais à priori ce n'est pas la bonne piste. 
Les param ont des valeurs par défaut :
dropbear.@dropbear[0]=dropbear
dropbear.@dropbear[0].PasswordAuth='on'
dropbear.@dropbear[0].RootPasswordAuth='on'
Par acquis de conscience, j'ai testé en basculant les états sur off, ça ne change rien (il faut rebooter OMR ou redémarrer un daemon pour la prise en compte des réglages ?). Ceci dit, dropbear, ce serait pour de l'accès par SSH ? Or ce qui ne fonctionne pas c'est l'authentification sur le portail web avec le compte root (http://192.168.100.1/cgi-bin/luci/) 
PS : aucune piste ne m'inspire non plus dans la config de luci

**Reply by Ysurac** [reply]
Luci access are set via rpcd config.

**Reply by Altarus1** [reply]
luci.sauth=internal
luci.sauth.sessionpath='/tmp/luci-sessions'
luci.sauth.sessiontime='3600'
luci.ccache=internal
luci.ccache.enable='1'
...
luci.apply=internal
luci.apply.rollback='90'
luci.apply.holdoff='4'
luci.apply.display='1.5'
luci.apply.timeout='20'


rpcd.@rpcd[0]=rpcd
rpcd.@rpcd[0].socket='/var/run/ubus/ubus.sock'
rpcd.@rpcd[0].timeout='120'
rpcd.@login[0]=login
rpcd.@login[0].username='root'
rpcd.@login[0].password='$p$root'
rpcd.@login[0].read='*'
rpcd.@login[0].write='*'

la page d'authentification affiche le message "Nom d'utilisateur et/ou mot de passe invalides ! Réessayez." dans le cas où un mauvais mdp est saisi. Rien ne s'affiche, mais rien ne se passe non plus quand le mdp est bon. C'est comme si le script du formulaire ne s'exécutait pas (ou bloque l'accès)

**Reply by Ysurac** [reply]
Pour "rpcd.@login[0].read" et "rpcd.@login[0].write", la valeur est bien '*' ? je ne vois pas l'étoile.
Si c'est le cas essaye de vider le cache de ton navigateur. Si pas de changement, vérifie que rpcd est bien lancé via ```ps w | grep rpcd```.

**Reply by Altarus1** [reply]
Oui, il y a bien les * dans la config, le copier/coller les a fait sauter. J'ai ouvert le portail dans le même navigateur en navigation privé, c'est ok l'authentification fonctionne, MERCI !! ça fonctionne aussi avec un autre navigateur, en revanche avec le même navigateur, le vidage du cache et l'actualisation forcée ne permet pas l'authentification. Mais cela confirme bien un soucis de cache, cookie ou autre coté navigateur donc pb résolu.

Si je retrouve l'option dans OMR qui m'a fait penser que le verrouillage venait d'elle, et si son rôle n'est toujours pas plus clair, si tu veux bien je reviendrais vers toi sur ce fil pour des explications.

Encore MERCI pour le support et MERCI également pour OMR que je découvre et qui boost vraiment ma connexion (agrégation 4G/ADSL)
